cli/commands/catalog/module/repo_test.go

```diff
@@ -0,0 +1,70 @@
+package module
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFindModules(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
+		repoPath        string
+		expectedModules Modules
+		expectedErr     error
+	}{
+		{
+			"testdata/repo1",
+			Modules{
+				&Module{
+					title:       "ALB Ingress Controller Module",
+					description: "This Terraform Module installs and configures the [AWS ALB Ingress Controller](https://github.com/kubernetes-sigs/aws-alb-ingress-controller) on an EKS cluster.",
+					url:         "https:/github.com/gruntwork-io/terraform-aws-eks/tree/master/modules/eks-alb-ingress-controller",
```

Comment:
1. These are private GH repos that most TG users won't have access to, so they wouldn't be able to run these tests. Can you figure out repos to test with that are publicly accessible? Or even add test fixtures to the TG repo itself?
1. Do these tests download this URL each time? If so, pin to a specific commit rather than `master`, as some day, we may change what's in the repo, and these tests will start failing.

Reply:
It uses fixtures located at `testdata/find_modules/terraform-aws-eks`. Therefore we don't need access to the remote repository.